package.json

```diff
@@ -85,7 +85,7 @@
         "watch:sass:min": "npm run build:sass:min -- --watch"
     },
     "engines": {
-        "node": ">=12"
+        "node": ">=18"
```

Comment:
Please could you drop this back to node 16 since I need to support this in a few environments - cheers!